### PR TITLE
 quicksort algo added

### PR DESCRIPTION
**Pull request by saumya1singh**

### CODE REVIEW

**Comment by saumya1singh** [comment]
@abhishekchopra13  added quicksort algorithm . Please review if it's the correct format .

File: quicksort.cpp [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+// A utility function to swap two elements 
+void swap(int* a, int* b) 
+{ 
+	int t = *a; 
+	*a = *b; 
+	*b = t; 
+} 
+
+/* This function takes last element as pivot, places 
+the pivot element at its correct position in sorted
```
Comment (Bhavnaharitsa) [inline quicksort.cpp:14]: push this purpose on top

File: quicksort.cpp [hunk 2]
```diff
@@ -0,0 +1,75 @@
+/* C++ implementation of QuickSort */
+#include <bits/stdc++.h> 
+using namespace std; 
+
+// A utility function to swap two elements 
+void swap(int* a, int* b)
```
Comment (Bhavnaharitsa) [inline quicksort.cpp:6]: add sample input-output

**Review by chanmol1999 — CHANGES_REQUESTED** [review]
You have to place this inside the algorithms folder.
No Response going to close

**Pull request closed without merging**